src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const listItems = list.querySelectorAll('li');
+
+listItems.forEach(li => {
```

Comment:
Consider adding a check to ensure that 'li' is not null or undefined before using it.